client/src/main/java/tds/student/Student.java

```diff
@@ -4,37 +4,36 @@
  * Represents a student taking an exam
  */
 public class Student {
-    private long id;
-    private String studentId;
-    private String stateCode;
-    private String clientName;
+    private final long id;
+    private final String externalStudentId;
```

Comment:
The reason I changed it was because it felt odd to me to have `student.id` and `student.studentId`.  If we can come up with something other than that I'm up for it.